content/kubermatic/master/guides/installation/add_seed_cluster/_index.en.md

```diff
@@ -174,14 +177,17 @@ watch kubectl -n kubermatic get pods
 
 ## Update DNS
 
-The apiservers of all user cluster control planes running in the seed cluster are exposed by the
-NodePort Proxy. By default each user cluster gets a virtual domain name like
+Depending on the chosen [Expose Strategy]({{< ref "../expose_strategy">}})), the control planes of all user clusters
+running in the Seed cluster will be exposed by the `nodeport-proxy` or using
+services of type `NodePort` directly.
+By default each user cluster gets a virtual domain name like
 `[cluster-id].[seed-name].[kubermatic-domain]`, e.g. `hdu328tr.kubermatic.kubermatic.example.com`
 for the Seed from the previous step when `kubermatic.example.com` is the main domain where the
```

Comment:
Not related to this PR, but it should be reformulated to something like `with kubermatic.example.com being the main domain where the KKP etc.`